rust/ql/src/queries/security/CWE-614/InsecureCookie.ql

```diff
@@ -39,9 +39,9 @@ module InsecureCookieConfig implements DataFlow::ConfigSig {
     node instanceof Sink
   }
 
-  predicate isBarrier(DataFlow::Node node) {
-    // setting the 'secure' attribute to true
-    cookieSetNode(node, "secure", true)
+  predicate isBarrierIn(DataFlow::Node node) {
+    // setting the 'secure' attribute
+    cookieSetNode(node, "secure", _)
```

Comment:
Is this change because setting a cookie's security to `false` is a sink, and hence we want to make `false` a barrier as well, as otherwise two sources would flow to a sink?

What is the `isBarrier` -> `isBarrierIn` change for?